pkg/controller/installation/products/rhsso/reconciler.go

```diff
@@ -101,8 +105,32 @@ func (r *Reconciler) handleAwaitingNSPhase() (v1alpha1.StatusPhase, error) {
 		return v1alpha1.PhaseFailed, err
 	}
 	if ns.Status.Phase == v1.NamespaceActive {
-		logrus.Infof("Creating subscription")
-		return v1alpha1.PhaseCreatingSubscription, nil
+		err := r.client.Create(context.TODO(), &v12.RoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "integreatly-operator-rolebinding",
+				Namespace: r.Config.GetNamespace(),
+				Labels: map[string]string{
+					"integreatly": "yes",
+				},
+			},
+			RoleRef: v12.RoleRef{
+				Name:     "admin",
+				Kind:     "ClusterRole",
+				APIGroup: "rbac.authorization.k8s.io",
+			},
+			Subjects: []v12.Subject{
+				{
+					Kind:      "ServiceAccount",
+					Name:      "integreatly-operator",
+					Namespace: "005-test",
```

Comment:
Should this be a var of some kind? Same above.

Reply:
Whoops! Yes it should